public/js/monitor.js

```diff
@@ -1,4 +1,56 @@
 "use strict";
+/* global ga */
+
+function sendEvent(category, action, label){
+  if(ga){
+  ga("send", "event", category, action, label);
+  }
+}
+
+function handleSignUpClicks(string){
+  const eventCategory = "Sign Ups";
+  const eventAction = "Clicked Sign Up";
+  if(window.location.href.indexOf("breach") > -1){
+    sendEvent(eventCategory, eventAction, "Sign up button clicked from featured breach.");
+  }
+  else if (document.getElementById("what-to-do")) {
+    sendEvent(eventCategory, eventAction, "Sign up button clicked after scanning email with breaches.");
+  }
+  else {
+    if(window.location.href.indexOf("scan") > -1){
+      sendEvent(eventCategory, eventAction, "Sign up button clicked after scanning email with no breaches.");
+    }
+    else {
+      sendEvent(eventCategory, eventAction, "Sign up button clicked from landing page.");
+    }
+  }
+}
+
+function handleEmailScans(){
+  const eventCategory = "Scans";
+  const eventAction = "Scanned Email Address";
+  if(window.location.href.indexOf("breach") > -1){
+    sendEvent(eventCategory, eventAction, "User submitted email from a featured breach.");
+  }
+  else if(document.getElementById("what-to-do")){
+    sendEvent(eventCategory, eventAction, "User submitted additional email.");
+  }
+  else {
+    sendEvent(eventCategory, eventAction, "User submitted email from homepage.");
+  }
+}
+
+function handlePageViews(){
+  const eventCategory = "Scan Results";
+  const eventAction = "Viewed Scan Results";
+  if(window.location.href.indexOf("scan") > -1){
```

Comment:
See https://github.com/mozilla/blurts-server/pull/190/files#r194951250 comment above.

Also, since we're duplicatin' the `?breach=` and `/scan` logic in 2 places, not sure if it's worthy of a helper function, a la `function isScanPage() {...}` and `function isBreachPage() {...}`. Maybe not, I'm just sayin'.

Reply:
I thought about this and if my recollection serves it done got convoluted on account of the event labeling. Will reinvestigate.